cms/templates/js/access-editor.underscore

```diff
@@ -0,0 +1,48 @@
+<form>
+    <% if (xblockInfo.get('prereqs').length > 0) { %>
+        <h3 class="modal-section-title"><%= gettext('Limit Access') %></h3>
+        <div class="modal-section-content gating-prereq">
+            <ul class="list-fields list-input">
+                <p class="field-message">
+                    <%- gettext('Select a prerequisite subsection and enter a minimum score percentage to gate this subsection.') %>
+                </p>
+                <li class="field field-select">
+                    <label for="prereq" class="label">
+                        <%- gettext('Prerequisite:') %>
+                    </label>
+                    <select id="prereq" class="input">
+                        <option value="">No prerequisite</option>
+                        <% _.each(xblockInfo.get('prereqs'), function(prereq){ %>
+                            <option value="<%- prereq.block_usage_key %>"><%- prereq.block_display_name %></option>
+                        <% }); %>
+                    </select>
+                </li>
+                <li id="prereq_min_score_input" class="field field-input input-cosmetic">
+                    <label for="prereq_min_score" class="label">
+                        <%- gettext('Minimum Score:') %>
+                    </label>
+                    <input type="text" id="prereq_min_score" name="prereq_min_score" class="input input-text" size="3" aria-describedby="Minimum score must be an integer between 0 and 100." placeholder="0 - 100" />
+                </li>
+                <div id="prereq_min_score_error" class="message-status error">
+                    <%- gettext('Minimum score must be an integer between 0 and 100.') %>
+                </div>
+            </ul>
+        </div>
+    <% } %>
+    <h3 class="modal-section-title"><%= gettext('Set as a Prerequisite') %></h3>
+    <div class="modal-section-content gating-is-prereq">
+        <ul class="list-fields list-input">
+            <li class="field field-checkbox checkbox-cosmetic">
+                <input type="checkbox" id="is_prereq" name="is_prereq" class="input input-checkbox" />
+                <label for="is_prereq" class="label">
+                    <i class="icon fa fa-check-square-o input-checkbox-checked"></i>
+                    <i class="icon fa fa-square-o input-checkbox-unchecked"></i>
+                    <%- gettext('Yes, set as a prerequisite') %>
```

Comment:
NIT: Change copy to "Yes, set this as a prerequisite other content can limit access with"